src/renderer/nav/Validator.tsx

```diff
@@ -38,7 +38,9 @@ const Validator = () => {
   const filterRef = useRef<HTMLInputElement>({} as HTMLInputElement);
   const validator = useAppSelector(selectValidatorNetworkState);
   const validatorImageName = 'cryptoworkbench/solana-amman';
-  const [validatorImageTag, setValidatorImageTag] = useState<string | null>('');
+  const [validatorImageTag, setValidatorImageTag] = useState<string | null>(
+    `${validatorImageName}:v1.10.31`
```

Comment:
horrible! and not worth changing for v0.4.0

if we're lucky, there's a simple API endpoint that tells solana validator's what the default mainnet version is, but i don't know of one atm -i don't want to parse github releases, discored etc :)

Reply:
see #235 :)

Reply:
> horrible! and not worth changing for v0.4.0

😆 

> if we're lucky, there's a simple API endpoint that tells solana validator's what the default mainnet version is, but i don't know of one atm -i don't want to parse github releases, discored etc :)

Yea, picking the right one feels tricky. e.g., the most common one on mainnet right now is 1.10.29, but the most recent 1.10.* release is 1.10.31 🤷